justfile

```diff
@@ -12,7 +12,7 @@ clippy:
 
 fmt:
   #rustup component add rustfmt --toolchain nightly
-  rustfmt +nightly --edition 2021 $(find . -type f -iname *.rs)
+  rustfmt +nightly --edition 2024 $(find . -type f -iname *.rs)
```

Comment:
need to port this to the `lint` and `rustfmt` workflows in .github/workflows also to get CI to validate it (right now it's failing)